lib/lita/handlers/puppet.rb

```diff
@@ -158,8 +158,9 @@ def nodes_with_class(response)
         end
 
         response.reply("#{username}, let me see what I can find in PuppetDB for you.")
-
-        puppet_classes = class_nodes(url, puppet_class)
+        
+        search = class_camel(puppet_class)
+        puppet_classes = class_nodes(url, search)
         if puppet_classes.empty?
           response.reply("There are no nodes with #{puppet_class} class, are you sure its a valid class?")
```

Comment:
I'm torn because the camel case is unique to searching within puppetdb. If I'm just thinking of a class name in puppet I think of the standard lowercase. Thinking aloud though, your change is more true to showing the user what was searched for.